submissions/LuckyDnepr/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles-menu.css" />
+    <title>HTML & CSS практика: Hooli-style Popup</title>
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <div class="logo">LOGO</div>
+        <nav class="navigation">
+          <a href="#Mail" class="nav_item">Mail</a>
+          <a href="#Images" class="nav_item">Images</a>
+          <input id="nav_popup_check" type="checkbox" />
+          <label for="nav_popup_check" id="nav_popup_label">
+            <img
+              class="nav_icons"
+              src="./icons/popup-button.png"
+              alt="popup-menu"
```

Comment:
[Read this article](https://htmlacademy.ru/blog/articles/alt-text) to find out how and when populating `alt` attribute

Reply:
I read the article. Focusing on using screen readers, I think I fixed the code in the part about setting the <alt> attribute.
Although without experience, some fixes may be unnecessary. Please comment on it.

Reply:
First of all, don't resolve comments., it's for reviewers)
Icon is a decorative image which means `alt` attribute must be empty.)

Reply:
I need to unresolved all comments?
About <alt> on icons, I will resolve this...

Reply:
About unresolved comments, right now all good, I mean that for future homeworks.

Reply:
W3C standard saying:
When an [a] element that creates a [hyperlink], or a [button] element, has no textual content but contains one or more images, the [alt] attributes must contain text that together convey the purpose of the link or button.
Other opinion in W3C:
Do not specify irrelevant alternate text when including images intended to format a page, for instance, alt="red ball" would be inappropriate for an image that adds a red ball for decorating a heading or paragraph. In such cases, the alternate text should be the empty string (""). Authors are in any case advised to avoid using images to format pages; style sheets should be used instead.

I tested my page in NVDA, and I think, my new changes are correct.
Please review.

Reply:
Impressive :open_mouth: 
In browser focus going to interactive elements, and we can implement some strategies:

1. Add `aria-label` attribute to the interactive element and the screen reader will read him;
2. Add text along with the image but hide him using `visually-hidden` utility class;

No matter what strategy you choose be sure it solves your current task.